libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -313,7 +313,13 @@ export class OperationExecutionManager {
       case OperationStatus.Failure: {
         // Failed operations get reported, even if silent.
         // Generally speaking, silent operations shouldn't be able to fail, so this is a safety measure.
-        const message: string | undefined = record.error?.message;
+        let message: string | undefined = undefined;
+        if (record.error) {
+          if (!(record.error instanceof AlreadyReportedError)) {
```

Comment:
@iclanton @dmichon-msft Any code that propagates errors needs to handle `AlreadyReportedError`.  Otherwise we get ugly output like this:

![image](https://github.com/microsoft/rushstack/assets/4673363/07350079-bb5d-4e89-b327-92193342ec6b)